src/libs/SidebarUtils.js

```diff
@@ -308,7 +309,6 @@ function getOptionData(reportID) {
         lastActorDetails = lastActorDisplayName
             ? {
                   displayName: lastActorDisplayName,
-                  login: report.lastActorEmail,
```

Comment:
I removed it because it didn't seem like we were using that login anywhere. All the lines that use the `lastActorDetails` below this don't seem to care about the login field, so it seemed fine to just remove it.